openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -41,19 +43,20 @@
     UserListingSerializer,
     UserSerializer
 )
-from .tasks import enroll_users
+from .tasks import bulk_user_registration, enroll_users
 from .utils import (
+    create_user,
     get_completed_course_count_filters,
     get_course_overview_same_org_filter,
     get_learners_filter,
     get_org_users_qs,
     get_roles_q_filters,
+    get_user_data_from_bulk_registration_file,
     get_user_org,
     get_user_org_filter,
     get_user_same_org_filter,
     is_courses_enroll_able,
-    is_courses_user_have_same_org,
-    send_registration_email
+    is_courses_user_have_same_org
```

Comment:
we should refactor this method name in near future, `do_courses_and_user_have_same_org`

Reply:
how about `is_courses_and_user_have_same_org`?